alacritty.yml

```diff
@@ -81,6 +81,18 @@
   # Set this to `None` to use the default theme variant.
   #gtk_theme_variant: None
 
+  # Power management mode
+  #
+  # This will control things like the GPU used to render Alacritty. When set to
+  # `Auto`, Alacritty will automatically switch to low power mode when on
+  # battery.
+  #
+  # Values for `energy`:
+  #   - Performance
+  #   - Powersave
+  #   - Auto
+  #energy: Auto
```

Comment:
I don't really like the name `energy`, since the option is purely about picking GPU, maybe I'd call it `preferred_gpu`, with variants `Integrated`, `Dedicated`, and `Auto`, or `performance_mode` with the variants you have now.

Reply:
> I don't really like the name energy, since the option is purely about picking GPU

I'm not a big fan of the name `energy` either, but I don't see why we would make an option exclusive to GPUs when there could be different purposes for it in the future. The goal is not to pick a specific GPU, but to select how power hungry Alacritty is. Thus it seems smarter to represent the goal of the user rather than the means for achieving them. So `preferred_gpu` would be a bad idea imho.

> or performance_mode with the variants you have now.

I've considered something like this, but `performance_mode: performance` doesn't make much sense to me. It's the power management mode Alacritty will be running in, that's the goal. The *performance* mode should always be performance, this is about energy not performance.

Reply:
you may call it powersaving_mode or something like that, but `energy: Performance` doesn't sound right to me.

Reply:
I've considered that option too, but I like none of the suggestions tbh. I don't like `energy` either, but `powersaving_mode: Performance` just doesn't make a ton of sense to me. Seems like that would be more of a boolean option.

Reply:
`hardware_acceleration_mode`, since it's basically what it's doing. Or maybe without `hardware`.